wix-ecom-backend/wix-ecom-backend/guides/1-wix-ecom-backend-intro.md

```diff
@@ -3,27 +3,69 @@ title: Introduction
+| [`onOrderRefunded()`](https://www.wix.com/velo/reference/wix-stores-backend/events/onorderrefunded) | [`onOrderTransactionsUpdated()`](https://www.wix.com/velo/reference/wix-ecom-backend/events/onordertransactionsupdatedd) |
+
+## Terminology
+
++ **Catalog:** A repository containing items for sale. A catalog can contain anything sellable, including physical products, services, gift cards, pricing plans, and custom items created for individual transactions like specialized project work. Wix business solutions, such as Wix Bookings and Wix Stores, provide built-in catalogs. External business solutions can provide the Wix eCommerce platform with access to their catalogs via the [Catalog SPI](https://www.wix.com/velo/reference/spis/wix-ecom/ecom-catalog/introduction). This enables robust integration of the platform’s cart and checkout functionalities with diverse business types.
++ **Cart:** Once customers have found catalog items they want to purchase, they can add these items to their cart. A cart holds information about a potential transaction, including details about selected items, prices, and discounts, as well as the potential buyer. Site visitors can see their cart on the cart page. Developers can access and manage a customer's cart with the [Cart API](https://www.wix.com/velo/reference/wix-ecom-backend/cart/introduction). Some purchasing flows bypass the cart entirely and proceed directly to checkout.
++ **Current Cart:** The [Current Cart API](https://www.wix.com/velo/reference/wix-ecom-backend/currentcart/introduction) automatically interacts with the site visitor's cart that is currently being used. This means you can use the same functionality as the Cart API, but without the need for a cart ID.
++ **Checkout:** Checkout is the page where a buyer finalizes a purchase. Each checkout holds information about the items to be purchased, price and tax summaries, shipping and billing information, any applied discounts, and more. Developers can access and manage checkout details and trigger checkout-related events with the [Checkout API](https://www.wix.com/velo/reference/wix-ecom-backend/checkout/introduction).
++ **Order:** Once a customer has committed to a purchase, an order is created. An order holds information about purchased items, price and tax summaries, shipping and billing information, any applied discounts, and the status of payment and fulfillment. In the dashboard, business staff can create new orders, view and edit existing orders, track fulfillment, and manage the payments cycle. The [Orders API](https://www.wix.com/velo/reference/wix-ecom-backend/orders/introduction) enables apps or site owners to customize management of the order lifecycle, including viewing, editing, approving, canceling, and charging.
 
 ## Import Statement
 
 To use the wix-ecom-backend API, import `wixEcomBackend` from the `wix-ecom-backend` module:
 
 ```javascript
 import wixEcomBackend from 'wix-ecom-backend';
-```
+```
```

Comment:
I think this is meant to be near the top, without a heading. See here: https://musical-system-74556837.pages.github.io/style-guide/api-documentation-guidelines/intro-articles/main-intro-article#velo-import-snippets